### PR TITLE
297: Add & enable Silta default Varnish setup, resolves #297.

### LINKED ISSUE #297: Add Silta varnish service as default
Request by @evilfurryone

### PR DESCRIPTION
**Pull request by tormi**
Resolves #297.

### CODE REVIEW

File: silta/silta-prod.yml [hunk 1]
```diff
@@ -64,3 +64,13 @@ mariadb:
 # Disable the built-in database when using an external database.
 #mariadb:
 #  enabled: false
+
+varnish:
+  enabled: true
+  resources:
```
Comment (tormi) [inline silta/silta-prod.yml:70]: Should we add limits as well?

Comment (misterjoonas) [reply silta/silta-prod.yml:70]: Yes, it's good to provide `limits` values also as an example. There should also be a comment that they need to be adjusted per project.
But what would be good or reasonable example values @evilfurryone if the file `storageBackend` is used with `512M` below?

Comment (tormi) [reply silta/silta-prod.yml:70]: > There should also be a comment that they need to be adjusted per project.

This applies to all services, not only Varnish.

Comment (tormi) [reply silta/silta-prod.yml:70]: Increased, 086a2d8.

**Review by misterjoonas — APPROVED** [review]
We can merge these Silta changes, varnish_purger can be added as soon as D10 compatibility is released https://www.drupal.org/project/varnish_purge/issues/3299406

**Pull request merged**